backend/src/openbeheer/types/_open_beheer.py

```diff
@@ -146,7 +150,22 @@ def as_ob_fieldtype(
             return OBFieldType.string
 
 
-def _fetch_procestype_options():
+def _cached[F: Callable[[], object]](f: F) -> F:
+    """Caching decorator, used for caching responses for a day
+
+    Like functools cache/lru_cache adds a `clear_cache` method on the function
+    """
+    key = f.__qualname__
+    function: F = lambda: django_cache.get_or_set(  # type: ignore get_or_set annotation is bad
+        key, default=f, timeout=60 * 60 * 24
+    )
+    function.clear_cache = lambda: django_cache.delete(key)  # pyright: ignore[reportFunctionMemberAccess]
```

Comment:
Pulling the cache calls outside the body of the `options()` function makes it possible to clear them if needed.

Normally, as a user, I'd CTRL-F5 to tell my browser to suppress/set the appropriate cache control headers.